examples/chip-tool/commands/tests/TestCommand.h

```diff
@@ -92,14 +87,14 @@ class TestCommand : public CHIPCommand,
             status.mStatus == chip::Protocols::InteractionModel::Status::UnsupportedCommand;
     }
 
-    void Wait()
-    {
-        if (mDelayInMs.HasValue())
-        {
-            chip::test_utils::SleepMillis(mDelayInMs.Value());
-        }
-    };
-    chip::Optional<uint64_t> mDelayInMs;
     chip::Optional<char *> mPICSFilePath;
     chip::Optional<uint16_t> mTimeout;
+    std::map<std::string, ChipDevice *> mDevices;
+
+    // Prevent interaction model events to affects the current test status when set to false.
+    // It exists because if an errors happens while processing a response the allocated
+    // command client/sender (ReadClient/WriteClient/CommandSender) can not be deallocated
+    // as it still used by the stack afterward. So a task is scheduled to be runned to close the
+    // test suite as soon as possible, and pending events are ignored in between.
```

Comment:
```suggestion
    // When set to false, prevents interaction model events from affecting the current test status.
    // This flag exists because if an error happens while processing a response the allocated
    // command client/sender (ReadClient/WriteClient/CommandSender) can not be deallocated
    // as it still used by the stack afterward. So a task is scheduled to run to close the
    // test suite as soon as possible, and pending events are ignored in between.
```